### PR TITLE
Added Australia page

### PR DESCRIPTION
**Pull request by HPreast**
Added new html page for Australia

### CODE REVIEW

**Review by coltmay — APPROVED** [review]

File: australia.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <li><a href="#home">Home</a></li>
+            <li><a class="active" href="#australia">Australia</a></li>
+            <li><a href="#iceland">Iceland</a></li>
+            <li style="float:right"><a class="active" href="#italy">Italy</a></li>
+            <li><a href="#japan">Japan</a></li>
+        </ul>
+    </header>
+    <main>
+        <div>
+            <img src="ausLandscape.jpeg" href="Australian Landscape View">
```
Comment (coltmay) [inline australia.html]: Link to the images folder.
Use `alt` instead of `href`.

File: australia.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <link rel="stylesheet" href="./styles/main.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Australia</title>
+</head>
+<body>
+    <header>
+        <div>
+            <img src="img_australiaFlag.jpeg" href="Australian Flag">
```
Comment (EthanMathis) [inline australia.html]: your image source should look something like `src="images/img_australiaFlag.jpeg" alt="Autralian Flag"`

**Review by EthanMathis — CHANGES_REQUESTED** [review]

File: australia.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 <body>
     <header>
         <div>
-            <img src="img_australiaFlag.jpeg" href="Australian Flag">
+            <img src="images/img_australiaFlag.jpeg" href="Australian Flag">
```
Comment (EthanMathis) [inline australia.html]: href should be alt :)

File: australia.html [hunk 4]
```diff
@@ -22,7 +22,7 @@
     </header>
     <main>
         <div>
-            <img src="ausLandscape.jpeg" href="Australian Landscape View">
+            <img src="images/ausLandscape.jpeg" href="Australian Landscape View">
```
Comment (EthanMathis) [inline australia.html]: href should be alt :)

**Review by EthanMathis — CHANGES_REQUESTED** [review]

**Review by coltmay — APPROVED** [review]
Looks great!  Thanks!

**Review by EthanMathis — APPROVED** [review]

**Review by NSPatton — APPROVED** [review]

**Pull request merged**